external/wakaama/examples/shared/Make.defs

```diff
@@ -38,5 +38,9 @@
 CSRCS += commandline.c connection.c platform.c memtrace.c
 CFLAGS+=-I$(TOPDIR)/../external/wakaama/core
 
+ifeq ($(CONFIG_LWM2M_SECURITY),y)
```

Comment:
Hi.
Actually we made that configuration for supporting platform TLS(mbedTLS) in wakaama because we need to support tcp and tls. If it's name is confused, i will change the name.